bindings/python/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[package]
+name = "pyiceberg_core_rust"
+version = "0.0.1"
+edition = "2021"
```

Comment:
nit: 2024?

Reply:
Hi, @kevinjqliu, thanks a lot for the review! The "Rust edition" is the mechanism used to allow backwards-incompatible changes in Rust since version 1.0. The latest edition is 2021. The next one, 2024, has not been released yet. More information can be found at https://doc.rust-lang.org/edition-guide/rust-2024/index.html.